frontend/src/metabase/query_builder/components/view/sidebars/SummarizeSidebar/BreakoutColumnList/BreakoutColumnListItem/BreakoutColumnListItem.styled.tsx

```diff
@@ -12,7 +12,8 @@ export const Content = styled.div`
   align-items: center;
   border-radius: 6px;
 
-  ${BucketPickerPopover.TriggerButton} {
+  .${BucketPickerPopoverStyle.triggerButton} {
```

Comment:
I didn't verify every class, but the sidebar should have this binned button visible even without hovering.

<img width="472" alt="Screenshot 2025-02-24 at 2 05 46 PM" src="https://github.com/user-attachments/assets/ae1fa9bb-cdd7-4be8-aabd-8a052330e7e3" />